ui/src/app/components/pods/pods.tsx

```diff
@@ -107,6 +110,16 @@ export const ReplicaSet = (props: {rs: RolloutReplicaSetInfo; showRevision?: boo
                 <ThemeDiv className='pods__header'>
                     <span style={{marginRight: '5px'}}>{rsName}</span> <ReplicaSetStatusIcon status={props.rs.status as ReplicaSetStatus} />
                     {props.showRevision && <div style={{marginLeft: 'auto'}}>Revision {props.rs.revision}</div>}
+                    {props.rs.scaleDownDeadline && (
+                        <div style={{marginLeft: 'auto'}}>
+                            <Ticker>
+                                {(now) => {
+                                    const time = moment(props.rs.scaleDownDeadline).diff(now, 'second');
+                                    return time === 0 ? null : <ActionButton label={`delay: ${time}s`} />;
```

Comment:
I don't think it makes sense to use `ActionButton` here because user cannot really click anything and trigger any action. Can we use info item instead:

![image](https://user-images.githubusercontent.com/426437/152232540-d771e6d6-d8d1-42f4-b4e7-ca50a75af19c.png)

revision.tsx
```typescript
const ImageItems = (props: {images: ImageInfo[]}) => {
    return (
        <div>
            {props.images.map((img) => {
                let imageItems = img?.tags?.map((t) => {
                    return {content: t, icon: IconForTag(t)} as InfoItemProps;
                });
                if (imageItems.length === 0) {
                    imageItems = [];
                }
                imageItems.push({content: <Tooltip content='Scaledown in 3 hours 5 mins'><span>3 hours 5 mins</span></Tooltip>, icon: 'fa fa-clock'} as any);
                return <InfoItemRow key={img.image} label={<ThemeDiv className={`image image--${img.color || 'unknown'}`}>{img.image}</ThemeDiv>} items={imageItems} />;
            })}
        </div>
    );
};

```